frontend/src/app/on-time/on-time-grid/on-time-grid.component.ts

```diff
@@ -387,6 +387,7 @@ export class OnTimeGridComponent<TData extends AbstractPerformance> {
     const lateRatio = sumBy(value, "lateRatio");
     const onTimeRatio = sumBy(value, "onTimeRatio");
```

Comment:
Maybe we could call these variables onTimeRatioSum etc.? Just to make it a bit more explicit from looking at the variable names.